### PR TITLE
Automatically update in builded app.

### PR DESCRIPTION
**Pull request by HyunmoAhn**
**Related Issue**
#67 

## Notice
- occur dependency about `electron-updater`.
- When build for mac, If config is `target: dmg`, It doesn't update automatically. So config must be set `target: dmg+zip` and It is default configure. [Show Related issues](https://github.com/electron-userland/electron-builder/issues/2137)
- I test about Mac OS and Window x64 in my forked repository. It worked well. but doesn't occur notification in Window OS.

### CODE REVIEW

File: public/electron.js [hunk 1]
```diff
@@ -81,10 +82,24 @@ function disableDetachedMode() {
   mainWindow && mainWindow.setIgnoreMouseEvents(false);
 }
 
+function checkAndDownloadUpdate() {
+  autoUpdater.setFeedURL({
+    provider: 'github',
+    owner: 'kamranahmedse',
+    protocol: 'https',
+    repo: 'pennywise',
+  });
```
Comment (kamranahmedse) [inline public/electron.js]: This is not required. [According to the docs](https://www.electron.build/auto-update#quick-setup-guide):

> Do not call setFeedURL. `electron-builder` automatically creates `app-update.yml` file for you on build in the resources (this file is internal, you don’t need to be aware of it).

Comment (HyunmoAhn) [reply public/electron.js]: OK. Thank you. I missed that. 😢 

I delete unnecessary code.

**Comment by HyunmoAhn** [comment]
@The-Code-Monkey 
I have never thought about sequence of `createWindow` and `checkAndDownloadUpdate`.
I think that which is called first is not important.
Why do you think `checkAndDownloadUpdate` should be called first?

**Comment by kamranahmedse** [comment]
Thanks

**Pull request merged**